# Message to request different follow behavior

Header header

bool Ball
bool Goal

int32 BallCenterX
int32 BallCenterY 
int32 BallRadius

int32 GoalX
int32 GoalY
int32 GoalW
int32 GoalH

float32 DistBall
float32 DistGoal
